ginga/web/jupyterw/ImageViewJpw.py

```diff
@@ -46,36 +46,43 @@
 """
 from ipyevents import Event as EventListener
 
-from ginga import AstroImage
+from ginga import ImageView, AstroImage
 from ginga import Mixins, Bindings
+from ginga.canvas import render
 from ginga.util.toolbox import ModeIndicator
 
-try:
-    # See if we have aggdraw module--best choice
-    from ginga.aggw.ImageViewAgg import ImageViewAgg as ImageView
+from ginga.web.jupyterw import JpHelp
 
-except ImportError:
-    # fall back to pillow if aggdraw not available
-    from ginga.pilw.ImageViewPil import ImageViewPil as ImageView
 
-from ginga.web.jupyterw import JpHelp
+class ImageViewJpwError(ImageView.ImageViewError):
+    pass
 
 
-class ImageViewJpw(ImageView):
+class ImageViewJpw(ImageView.ImageViewBase):
 
     def __init__(self, logger=None, rgbmap=None, settings=None):
-        ImageView.__init__(self, logger=logger,
-                           rgbmap=rgbmap,
-                           settings=settings)
+        ImageView.ImageViewBase.__init__(self, logger=logger,
+                                         rgbmap=rgbmap,
+                                         settings=settings)
 
+        self.t_.set_defaults(renderer='cairo')
+
+        self.rgb_order = 'RGBA'
```

Comment:
You had a comment before that said `# Format 'png' is ok with 'RGBA', but 'jpeg' only works with 'RGB'`. Is that no longer the case?

Reply:
This is now handled correctly in the `ginga.pilw.CanvasRenderPil` module, and we no longer need the comment or the restriction.